src/util.rs

```diff
@@ -74,7 +72,6 @@ impl<T> Default for Phantom<T> {
     }
 }
 
-#[allow(new_without_default)]
```

Comment:
Why did you remove that? Clippy keeps bothering me about the absence of a `Default`, even though there is one.